src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1733,6 +1734,103 @@ <h2 class="text--center">Balance sheet</h2>
               </tr>
               {{end}}
 
+              <!-- BALANCE SHEET - CURRENT ASSETS -->
+
+              {{if $currentAssets := $balanceSheet.current_assets}}
+
+                <tr>
+                  <th class = "strong" id="current-assets-heading">Current assets</th>
+                </tr>
+
+                <!-- BALANCE SHEET - CURRENT ASSETS - STOCKS -->
+
+              {{if $stocks := $currentAssets.stocks}}
+                <tr class="figures">
+                  <td id="stocks-label">Stocks:</td>
+
+                  <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
+
+                  <td id="stocks-curr-val" class="strong figure">
+                    {{if isNotEmpty $stocks.current_amount}}
+                      <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalInventories" format="ixt2:numdotdecimal">{{$stocks.current_amount}}</ix:nonFraction>
+                    {{end}}
+                  </td>
+
+                  <td id="stocks-prev-val" class="figure">
```

Comment:
Same here, Id's mistmatched:
Ours: `<td id="stocks-prev-val" class="figure">`
SmallFullAccountsModifiediXBRL: `<td id="ac50-prev-val" class="figure">`

Reply:
Carl palfrey has confirmed that Ids do not need to match the prototype- along as they are unique